warehouse/templates/accounts/login.html

```diff
@@ -45,43 +45,11 @@ <h1 class="page-title">{% trans title=title %}Log in to {{ title }}{% endtrans %
         {% endif %}
 
         <div class="form-group">
-          <label for="username" class="form-group__label">
-            {% trans %}Username{% endtrans %}
-            {% if form.username.flags.required %}
-            <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
-            {% endif %}
-          </label>
-          {{ form.username(placeholder=gettext("Your username"), autocapitalize="off", autocomplete="username", spellcheck="false", required="required", class_="form-group__field", **{"aria-describedby":"username-errors"}) }}
-          <div id="username-errors">
-            {% if form.username.errors %}
-            <ul class="form-errors" role="alert">
-              {% for error in form.username.errors %}
-              <li>{{ error }}</li>
-              {% endfor %}
-            </ul>
-            {% endif %}
-          </div>
+          {{ input_field("Username", "username", form.username, placeholder="Your username", autocomplete="username") }}
```

Comment:
Can you update this (and other similar places) so that it always uses the keyword-argument names for all fields? E.g.:

```suggestion
          {{ input_field(name="username", user_friendly_name="Username", form_instance=form.username, placeholder="Your username", autocomplete="username") }}
```